README.md

```diff
@@ -51,18 +51,17 @@ constants.RID = RID; // IMPORTANT: Without this all castToRID will do is convert
-| `optional`                                    | `optionalProperties(modelName: string)`                                                               |                                                          |
-| `getActiveProperties()`                       | `activeProperties(modelName: string)`                                                                 |                                                          |
-| `inherits`                                    | `ancestors(modelName: string)`                                                                        |                                                          |
-| `subclasses: ClassModel[]`                    | `children(modelName: string): string[]`                                                               |                                                          |
-| `descendantTree(boolean): ClassModel[]`       | `descendants(modelName: string, opt: { excludeAbstract?: boolean, includeSelf?: boolean }): string[]` | must be called with includeSelf=true to match v3 edition |
-| `queryProperties: Property[]`                 | `queryableProperties(modelName: string): PropertyDefinition[]`                                        |                                                          |
-| `inheritsProperty(propName: string)`          | `inheritsProperty(modelName: string, propName: string)`                                               |                                                          |
-| `toJSON`                                      | N/A                                                                                                   |                                                          |
-| `formatRecord(record: GraphRecord, opt = {})` | `formatRecord(modelName: string, record: GraphRecord, opt = {})`                                      |                                                          |
-
-PropertyDefinition objects also now store the class name under linkedClass as a string rather than a reference to the actual ClassDefinition instance
+| v3                                                       | v4 equivalent                                                                                                          | Notes                                                    |
+| -------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| `ClassModel.properties`                                  | `SchemaDefinition.getProperties(modelName: string)`                                                                    |                                                          |
```

Comment:
it seems like `ClassDefinition.properties` is still a thing, is that different than using `SchemaDefinition.getProperties(modelName: string)`?

Reply:
I will add a note for this

Reply:
https://github.com/bcgsc/pori_graphkb_schema/pull/20/commits/a710e41f1b6bc05522535b5508461941e6185769